mkinitcpio

```diff
@@ -221,7 +221,7 @@ build_image() {
     # If this pipeline changes, |pipeprogs| below needs to be updated as well.
     find . -mindepth 1 -printf '%P\0' |
             sort -z |
-            LANG=C bsdtar --null -cnf - -T - |
+            LANG=C bsdtar --uid 0 --gid 0 --null -cnf - -T - |
             LANG=C bsdtar --uid 0 --gid 0 --null -cf - --format=newc @- |
```

Comment:
dreisner suggested that this second line probably doesn't need the --uid/--gid, since it obviously doesn't work. should I remove those?

Reply:
Might want to let @esotericnonsense look over and ensure it doesn't break reproducability.

Reply:
> dreisner suggested that this second line probably doesn't need the --uid/--gid, since it obviously doesn't work. should I remove those?

This second line looks weird to me, indeed.

Reply:
> Might want to let @esotericnonsense look over and ensure it doesn't break reproducability.

I'm going to test it also. And is reproducibility :-)

Reply:
any news?

Reply:
btw, doesn't the above `sort -z` also need a `LANG=C` option (just in case)?

Reply:
@gdamjan
There are some more things to fix on that pipe, see #16. I'm doing a revamp of this pipeline some time this week, so you might need to rebase your patch after that.